cmd/revad/svcs/httpsvcs/ocdavsvc/put.go

```diff
@@ -182,27 +184,27 @@ func (s *svc) doPut(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	req2 := &storageproviderv0alphapb.InitiateFileUploadRequest{
+	uReq := &storageproviderv0alphapb.InitiateFileUploadRequest{
 		Ref: &storageproviderv0alphapb.Reference{
 			Spec: &storageproviderv0alphapb.Reference_Path{Path: fn},
 		},
 	}
 
 	// where to upload the file?
-	res2, err := client.InitiateFileUpload(ctx, req2)
+	uRes, err := client.InitiateFileUpload(ctx, uReq)
 	if err != nil {
 		logger.Error(ctx, err)
 		w.WriteHeader(http.StatusInternalServerError)
 		return
 	}
 
-	if res.Status.Code != rpcpb.Code_CODE_OK {
-		logger.Println(ctx, res.Status)
+	if uRes.Status.Code != rpcpb.Code_CODE_OK {
+		logger.Println(ctx, uRes.Status)
```

Comment:
this was using the `res` from the previous request, which is wrong. I prefixed the variables for clarity and to prevent these copy paste errors.